src/psycopmlutils/loaders/raw/load_coercion.py

```diff
@@ -13,19 +13,19 @@ def coercion(
         reason_for_coercion: Optional[str] = None,
         n: Optional[int] = None,
     ) -> pd.DataFrame:
-        """Load coarcion data.
+        """Load coercion data. Defaults to entire coercion data view.
```

Comment:
Also add this at the end of the "n" argument – it's not intuitive what `n  = None` returns. If optional keyword arguments aren't specified, they return None.